.github/workflows/flake-detector.yml

```diff
@@ -32,10 +32,10 @@ jobs:
         run: go mod download
 
       - name: Test all except caching parallely
-        run: CGO_ENABLED=0 go test  -count 20 -skip `cat flaky_tests.lst | go run tools/scripts/skip_tests/main.go` `go list ./... | grep -v internal/cache/...`
+        run: CGO_ENABLED=0 go test -timeout 75m -count 20 -skip `cat flaky_tests.lst | go run tools/scripts/skip_tests/main.go` `go list ./... | grep -v internal/cache/...`
```

Comment:
Does it really take 75m ? 
(Ideally, it should be %20 or similar higher than the actual run time, so that developers try to parallelize/optimize when writing new tests to fit into that run time and be conscious of increasing it)

Reply:
I'd like to increase the number of iterations as much as possibledv cv v cxvcxvxczvz
++  3 that It doesn't really. But here, we are already enforcing a top-level timeout.